crates/virtio-queue/src/lib.rs

```diff
@@ -1182,12 +1182,38 @@ mod tests {
             assert_eq!(desc.len(), 0x1000);
             assert_eq!(desc.flags(), VIRTQ_DESC_F_NEXT);
             assert_eq!(desc.next(), 1);
+            assert_eq!(c.ttl, c.queue_size - 1);
 
             assert!(c.next().is_some());
+            // The descriptor above was the last from the chain, so `ttl` should be 0 now.
+            assert_eq!(c.ttl, 0);
             assert!(c.next().is_none());
+            assert_eq!(c.ttl, 0);
         }
     }
 
+    #[test]
+    fn test_ttl_wrap_around() {
```

Comment:
I am not sure why is this called `wrap_around`. Can you add some more details about the test case that you're adding here?

Reply:
Added some comments in the test.